api/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "port": 3000,
+    "fuelDataPath": "bin/FuelData.txt"
+}
```

Comment:
A w JSONach na końcu nie powinno być psacji?

Reply:
Chyba trzeba, ale nie będę robił update żeby tylko to poprawić.